component/README.md

```diff
@@ -0,0 +1,16 @@
+# ocm - command line interface
+
+This folder contains an example how to build a component version for the OCM cli.
```

Comment:
but here it's `OCM cli`

either or... controller or ocm-cli?

Reply:
Ah, Hilmar, you're the man. Gerald didn't notice. :D I'll fix it in a sec. :)

Reply:
https://github.com/open-component-model/ocm-controller/pull/467